src/qibo/models/encodings.py

```diff
@@ -470,3 +470,31 @@ def _parametrized_two_qubit_gate(gate, q0, q1, params=None):
         return gate(q0, q1, *params)
 
     return gate(q0, q1)
+
+
+def ghz_state(nqubits: int, **kwargs):
+    """Generates an :math:`n`-qubits Greenberger-Horne-Zeilinger (GHZ) state that takes the form
```

Comment:
```suggestion
    """Generates an :math:`n`-qubit Greenberger-Horne-Zeilinger (GHZ) state that takes the form
```

Reply:
@mho291 a tip: you can click the "Commit suggestion" button here to directly commit a suggestion to the code instead of changing things locally and then pushing it. It also helps to track which changes were already implemented.

Reply:
Thanks @renatomello ! I wanted to do extra checks for the documentation, so I did it locally and generated the html file. :)